{{cookiecutter.github_repository}}/settings/common.py

```diff
@@ -50,6 +51,10 @@
 # INSTALLED APPS CONFIGURATION
 # ==========================================================================
 
+# cors
+# --------------------------------------------------------------------------
+CORS_ORIGIN_WHITELIST = env.list('CORS_ORIGIN_WHITELIST', default=['localhost', 'localhost:8000'])
```

Comment:
I think it make sense to not have any valid option here. If somebody forgets to remove these values then `localhost` and `localhost:8000` remains become exposed.